wp-content/plugins/duracelltomi-google-tag-manager/admin/admin.php

```
@@ -0,0 +1,1387 @@
+	require_once dirname( __FILE__ ) . '/../integration/soundcloud.php';
+}
+
+/**
+ * Callback function for add_options_page(). Generates the GTM4WP plugin options page.
+ *
+ * @see https://developer.wordpress.org/reference/functions/add_options_page/
+ *
+ * @return void
+ */
+function gtm4wp_show_admin_page() {
+	global $gtp4wp_plugin_url;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$gtp4wp_plugin_url'. (undefined)

(UnusedLocalVariable)